pkgs/development/tools/sqlint/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, bundlerApp, fetchurl }:
+
+let
+  gemset = import ./gemset.nix;
+  pgQueryVersion = gemset.pg_query.version;
+  LIB_PG_QUERY_TAG = "10-1.0.1";
+  libpgQuerySrc = fetchurl {
+    url = "https://codeload.github.com/lfittl/libpg_query/tar.gz/${LIB_PG_QUERY_TAG}";
+    sha256 = "0m5jv134hgw2vcfkqlnw80fr3wmrdvgrvk1ndcx9s44bzi5nsp47";
+  };
+in bundlerApp {
+  pname = "sqlint";
+  gemdir = ./.;
+  exes = [ "sqlint" ];
+
+  gemConfig = {
+    pg_query = attrs: {
+      #patches = [./dir.patch];
+      postConfigure = ''
+        mkdir -p $out/${attrs.ruby.gemPath}/gems/pg_query-${pgQueryVersion}/ext/pg_query
+        cp ${libpgQuerySrc} $out/${attrs.ruby.gemPath}/gems/pg_query-${pgQueryVersion}/ext/pg_query/libpg_query.tar.gz
```

Comment:
libpg_query.tar.gz should be in place [here](https://github.com/lfittl/pg_query/blob/de2c9d1f61d7be1d2ba8be24bc495221cbc626a1/ext/pg_query/extconf.rb#L13) for the install phase, but it's not.